internal/evaluator/conftest_evaluator.go

```diff
@@ -160,6 +160,20 @@ func (c conftestEvaluator) Evaluate(ctx context.Context, inputs []string) ([]out
 
 	results = append(results, runResults...)
 
+	// Evaluate total successes, warnings, and failures. If all are 0, then
+	// we have effectively failed, because no tests were actually ran due to
+	// input error, etc.
+	var totalSuccesses, totalWarnings, totalFailures int
```

Comment:
nit: a single counter should be sufficient

Reply:
ack